src/mod/applications/mod_callcenter/mod_callcenter.c

```diff
@@ -40,6 +40,7 @@
 #define CC_SQLITE_DB_NAME "callcenter"
 #define CC_APP_KEY "mod_callcenter"
 
+#define CC_OFFERED_AGENT_LIST_SIZE 10000
```

Comment:
@surya4  is it not too big?

Reply:
done @royalpinto reduced it to 2048 now so it can handle approx 128 agents in one level, if more required we can increase list size - '000000@default' -> 1+6+1+7+1=16 characters